R/bs_sass.R

```diff
@@ -61,23 +61,34 @@ bootstrap <- function(theme = bs_theme_get(),
   )
   opts <- utils::modifyList(opts, options)
 
-  # Temp dir for building the HTML dependencies
-  output_path <- tempfile("bscustom")
-  dir.create(output_path)
+  cache_key <- sass::sass_cache_key(theme, opts)
```

Comment:
We'll should incorporate the bootstrap version somehow here (which is determined below by `if (version %in% "3") version_bs3 else version_bs4`)

Reply:
Sounds good -- I've extracted this out into a function.